caffe2/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include(../cmake/Codegen.cmake)
 
 # ---[ ATen build
 if (INTERN_BUILD_ATEN_OPS)
+  message(WARNING "NOT only building core")
```

Comment:
What does it mean to "NOT only [build] core"?

Reply:
This is old from greg's patch, it can be deleted